.github/workflows/pull_request.yaml

```
@@ -15,4 +15,17 @@ jobs:
       - uses: DeterminateSystems/magic-nix-cache-action@main
         with:
           upstream-cache: https://ngi.cachix.org/
+      - id: flake-show
+        name: Obtain `nix flake show` output
+        run: |
+          sudo apt install colorized-logs
+          echo 'output<<EOF' >> $GITHUB_OUTPUT
+          nix flake show --all-systems | ansi2txt >> $GITHUB_OUTPUT
+          echo 'EOF' >> $GITHUB_OUTPUT
```

```suggestion
          echo 'output <<EOF' >> $GITHUB_OUTPUT
          nix flake show --all-systems | nix run nixpkgs#ansifilter >> $GITHUB_OUTPUT
          echo 'EOF' >> $GITHUB_OUTPUT
```

Not sure whether the reference to Nixpkgs will work like that...